test/jdk/java/awt/print/PrinterJob/PageDialogMarginTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2021,  Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2016, 2021, Oracle and/or its affiliates. All rights reserved.
```
Two many spaces, there should be one space after the comma.